modules/cbelasticsearch/models/JestClient.cfc

```diff
@@ -733,7 +733,7 @@ component
 	*
 	* @returns  any 	A CFML representation of the result.  If `returnObject` is flagged, will return the client JestResult
 	**/
-	private any function execute( required any action, returnObject=false ){
+	public any function execute( required any action, returnObject=false ){
```

Comment:
Since the arguments and code used in this method are Jest-specific, we can’t interface this method.

I’m OK opening this up for development, but any code we write which uses it, should be implemented in to an interfaces method so that the main client can make use of it, within the module conventions